Instruments/power_control.py

```diff
@@ -61,21 +61,19 @@ def get(self, n_slow_tries = 0, slow_wait = 0.5):
         if not success: raise ValueError("no response after 30 tries!!")
         return data
     def get_bytes(self,ending):
```

Comment:
you need to include a docstring here.  Also, you should include an example to test this -- what is the simplest possible function that you can currently write that reproduces the problem that you are currently having?

Logistically, it would make sense to make a new pull request that can currently reproduce the problem.

Reply:
added docstring to get_bytes and made pull request in spincore_apps

Reply:
current version has no docstring -- may have gotten lost w/ commenting out of blocks of code.